runtime/moonbase/src/precompiles.rs

```diff
@@ -115,6 +115,18 @@ where
 	R: pallet_evm::Config,
 {
 	fn execute(&self, handle: &mut impl PrecompileHandle) -> Option<PrecompileResult> {
+		// Filter known precompile addresses except Ethereum officials
+		if self.is_precompile(handle.code_address())
+			&& handle.code_address() > hash(9)
+			&& handle.context().address != address
```

Comment:
```suggestion
			&& handle.context().address != handle.code_address()
```
or add `let address = handle.code_address()` before